backend/configs/usr/lib/cgi-bin/factory_reset.py

```
@@ -0,0 +1,51 @@
+
+def _error(msg=""):
+    sys.stdout.write("Status: 400 Bad Request\r\n\r\nBad Request: %s" % msg)
+    sys.exit(1)
+
+
+def _die(msg=""):
+    sys.stdout.write("Status: 500 Internal Server Error\r\n\r\nInternal Server Error: %s" % msg)
+    sys.exit(1)
+
+
+def check_for_form(form):
```

Ruff (0.8.2)

Missing return type annotation for public function `check_for_form`

Add return type annotation: `None`

(ANN201)